bson/src/main/org/bson/json/JsonWriterSettings.java

```diff
@@ -552,6 +563,10 @@ public int getMaxLength() {
         return javaScriptConverter;
     }
 
+    Converter<BsonDbPointer> getBsonDbPointerConverter() {
```

Comment:
I opted to keep with the past and not to expose the converter to the public or allow them to customize it.

Reply:
Ack